__tests__/constants/default-processor-options.ts

```diff
@@ -46,5 +46,8 @@ export const DefaultProcessorOptions: IIssuesProcessorOptions = Object.freeze({
   exemptAllPrAssignees: undefined,
   enableStatistics: true,
   labelsToRemoveWhenUnstale: '',
-  labelsToAddWhenUnstale: ''
+  labelsToAddWhenUnstale: '',
+  activitiesResetStale: true,
```

Comment:
do you think a name like `ignoreUpdates: false`, `ignoreIssueUpdates: false`, `ignorePrUpdates: false` would read  more intuitively? "activity" is not necessarily an expected term IMO

Reply:
Yeah sure

Reply:
@luketomlinson done